### ISSUE TITLE
A11y_.NETOnlineREPL_.NET Editor_Screenreader: NVDA is not announcing the line number of the code while navigating using down arrow key.

### ISSUE THREAD

**Issue by pranshu512** [dotnet/try#915]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_.NET Online REPL_Web_Feb2021;#.NET Online REPL;#Win10;#FTP;#DesktopWeb;#WCAG1.3.1;#Screenreader;#Narrator;#NVDA;#Chrome;#ChromiumEdge;#Element:Text;

### **Actual:**
NVDA is not announcing the line number of the code while navigating using down arrow key.

**Same issue is also observed with Chromium Edge +Narrator.**

### **Expected:**
NVDA should announce the line number of the code while navigating using down arrow key.

### **Environment Details:**
Application Name: .NET Online REPL
Operating System: Windows10
Browser Version : Google Chrome Version 87.0.4280.141 (Official Build) (64-bit)
Chromium Edge Version 89.0.774.14 (Official build) dev (64-bit)
Screenreader: NVDA 2020.3,Narrator

### **Repro Steps:**

1. Open .NET Editor coding screen .
2. Start NVDA and navigate to .NET editor section with the help of down arrow key.
3.. Check whether NVDA announces the line number of the code while navigating using down arrow key.

### **User Impact:**
Screen reader user will not get  line number of the code.

**Comment by nelsondani** [comment]
30.000`[T]() @nnelsondani #35.000
Duplicate of #

>

**Comment by fsteffi** [comment]
still reproduces;https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

**Comment by AnKushSingh05** [comment]
#Closed;
Verified the issue in below environment. Issue is fixed hence closing the bug.
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha
We can use the CTRL+G shortcut keys to know about the current line. Hence closing the bug.

Uploading 915_Fixed.mp4…
